mindjunkies/forums/tests/test_views.py

```
@@ -0,0 +1,79 @@
+        self.comment = ForumComment.objects.create(topic=self.forum_topic, author=self.user, content='Test Comment')
+        self.reply = Reply.objects.create(parent_comment=self.comment, author=self.user, body='Test Reply')
+
+    def test_forum_home_view(self):
+        self.client.login(username='testuser', password='password')
+        response = self.client.get(reverse('forum_home', kwargs={'course_slug': self.course.slug}))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'forums/forum_home.html')
+
+    def test_forum_thread_view(self):
+        self.client.login(username='testuser', password='password')
+        response = self.client.get(reverse('forum_thread', kwargs={'course_slug': self.course.slug, 'module_id': self.module.id}))
```

Ruff (0.8.2)

Line too long (130 > 119)

(E501)